core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.scss

```diff
@@ -0,0 +1,73 @@
+@use "variables" as *;
+@import "mixins";
+
+:host {
+    display: flex;
+    align-items: center;
+    height: 100%;
+
+    ::ng-deep {
+        .p-button-tabbed {
+            box-shadow: none;
+            height: 100%;
+            margin-right: $spacing-4;
+        }
+
+        .p-selectbutton {
+            height: 100%;
+            display: flex;
+
+            .p-button-label {
+                color: $black;
+            }
+        }
+    }
+}
+
+dot-edit-page-lock-info {
+    margin-left: $spacing-1;
+}
+
+p-inputSwitch {
+    ::ng-deep {
+        .p-inputswitch-slider:after {
+            @include md-icon;
+            color: $color-palette-primary;
+            content: "lock_open";
+            font-size: 13px;
```

Comment:
use scss variables

Reply:
I think, might be wrong... that is because is the icon but we can use rem but better see how can we do primeicons if possible.